### DISCUSSION TITLE
How to build images locally ?

### DISCUSSION THREAD

**Discussion by atibdialpad** [open-telemetry/opentelemetry-collector-contrib#13618] [Q&A] [answered]
How do I build a local image ? Have added some debug logging in the code e.g printing the metrics name here https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/718956aaa4d59a76b05eafcebf0fc923b3edb74b/pkg/translator/prometheusremotewrite/metrics_to_prw.go#L66

**Comment by atibdialpad** [comment, marked as answer]
make docker-otelcontribcol

**Reply by MB175** [reply]
Using the make docker command leads to the following issue on my Darwin machine

```
github.com/ClickHouse/ch-go/proto
../../go/pkg/mod/github.com/!click!house/ch-go@v0.47.3/proto/col_map.go:124:11: internal compiler error: assertion failed

Please file a bug report including a short program that triggers the error.
https://go.dev/issue/new
make[2]: *** [otelcontribcol] Error 2
make[1]: *** [docker-component] Error 2
make: *** [docker-otelcontribcol] Error 2

```

**Reply by MB175** [reply]
upgrading to go1.19.1 darwin/amd64 does resolve this issue. See [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14455)
